src/ripple/overlay/impl/PeerSet.cpp

```diff
@@ -62,7 +59,7 @@ bool PeerSet::insert (std::shared_ptr<Peer> const& ptr)
 
 void PeerSet::setTimer ()
 {
-    mTimer.expires_from_now(mTimerInterval);
+    mTimer.expires_after(mTimerInterval);
```

Comment:
`expires_from_now` is [deprecated since at least 1.66](https://www.boost.org/doc/libs/1_66_0/doc/html/boost_asio/reference/basic_waitable_timer/expires_from_now.html), and the suggested alternative is `expires_after`.